tests/http/test_server.py

```diff
@@ -334,5 +345,105 @@ def test_without_slash_should_redirect(self):
     def test_can_wrap_wsgi_apps(self):
         response = self.fetch("/wsgi/", method="GET")
 
-        assert 200 == response.code
-        assert "Hello, world!" in tornado.escape.to_unicode(response.body)
+        self.assertEqual(200, response.code)
+        self.assertIn("Hello, world!", tornado.escape.to_unicode(response.body))
```

Comment:
Looks like a merge with our recent rewrite from unittest assertions to pytest assertions went bad here. Please use the `assert ...` style.

Reply:
Fixed